### ISSUE TITLE
Add install of python requirements

### ISSUE THREAD

**Issue by thegecko** [ARMmbed/mbed-js-gulp#1]
Currently the user needs to install the python requirements after cloning the jerryscript repo:

```bash
pip install -r build/jerryscript/targets/mbedos5/tools/requirements.txt
```

Can this be done automagically?

**Comment by matthewelse** [comment]
We could put it in tmpl/Makefile.tmpl?

**Comment by thegecko** [comment]
Sure

**Comment by thegecko** [comment]
Actually, shouldn't this be done post-cloning Jerry? That way it's only run once rather than every build.

**Comment by matthewelse** [comment]
Yeah you're right - I'll change it in main.js

**Comment by matthewelse** [comment]
Fixed in fc437b6

**Comment by matthewelse** [comment]
The only slight problem arises when you're installing pip modules globally (using sudo), though you can get around this by just running gulp as root the first time you run it.
